.github/workflows/build_and_test.yml

```diff
@@ -421,7 +395,7 @@ jobs:
         rm -rf ~/.m2/repository/org/apache/spark
```

Comment:
For `Java 11 build`, it seems `mvn install` run but can we just do `mvn compile testCompile`? If we can do so, we can remove `.m2` repository here like other places. What do you think?

Reply:
That works to me.

Reply:
@sarutak and @HyukjinKwon . I'd like to keep this `install` test coverage. This is a contribution from @wangyum to check a valid error check from JDK8 age.

Please see e5c176a243b76b3953cc03b28e6c281658da93c8

Reply:
Do we have another new place to test `install`?

Reply:
Oh, sure. Then it makes sense to keep `install`. +1 for keeping it.

Reply:
Thank you! You can add some comments there if you want~

Reply:
Done

Reply:
O.K, make sense. Thanks @dongjoon-hyun .